.github/workflows/spread-large.yaml

```diff
@@ -0,0 +1,47 @@
+name: Spread (large)
+on:
+  pull_request:
+    types: [ labeled ]
+  schedule:
+    - cron: "0 2 * * 0,3" # run at 2 AM on Sundays and Wednesdays
+
+jobs:
+  snap-build:
+    if: ${{ github.event.label.name == 'run-large-spread' || github.eventname == 'schedule' }}
```

Comment:
I don't actually know if this works; the intent is to run the jobs if either the PR got the 'run-large-spread' label, or if the event was the schedule triggering

Reply:
it looks fine